### PR TITLE
Readme updates 

### PR DESCRIPTION
**Pull request by codepo8**
Fixed Markdown issues, removed visual HTML, added new sections for headless and network and new screenshots

### CODE REVIEW

**Comment by codepo8** [comment]
Had to change all instances to Visual Studio Code.

File: README.md [hunk 1]
```diff
@@ -1,46 +1,81 @@
...
+**Show the browser's Elements and Network tool inside the Visual Studio Code editor and use it to fix CSS issues with your site and inspect the Network.**
 
-A VS Code extension that allows you to use the browser's Elements and Network tool from within the editor. The DevTools will connect to an instance of Microsoft Edge giving you the ability to see the runtime HTML structure, alter layout, fix styling issues, and view network requests. All without leaving VS Code.
+A Visual Studio Code extension that allows you to use the browser's Elements and Network tool from within the editor. The DevTools will connect to an instance of Microsoft Edge giving you the ability to see the runtime HTML structure, alter layout, fix styling issues, and view network requests. All without leaving Visual Studio Code.
 
-**Note**: This extension only supports Microsoft Edge
+**Note**: This extension _only_ supports Microsoft Edge
 
-![Microsoft Edge Tools - Demo](demo.gif)
+![Microsoft Edge Tools - Demo](basic_usage.gif)
```
Comment (mliao95) [inline README.md:10]: Do we also want to show network tool functionality here or is the main focus of the extension the Elements panel?

**Review by mliao95 — COMMENTED** [review]
LGTM - just one comment and I'll approve

**Comment by codepo8** [comment]
This is part of the update and the screenshots

**Comment by mliao95** [comment]
Right.  My main question was about whether we want to include the network tool in the main demo at the top of the README.  I'd imagine that this demo should showcase the main purpose of the extension and the question is whether we want to include the Network tool in that.

File: README.md (hunk 1 above)
Comment (codepo8) [reply README.md:10]: That's only the intro one. As Network isn't enabled by default, this would be confusing.

**Comment by codepo8** [comment]
No, I don't think it should be part of the first GIF as it needs to be turned on to show up. By default it is turned off.

File: README.md [hunk 2]
```diff
@@ -1,46 +1,81 @@
 
-<h1 align="center">
-  <br>
-  VS Code - Microsoft Edge Tools
-  <br>
-</h1>
+# Microsoft Edge Developer Tools for Visual Studio Code
 
-<h4 align="center">Show the browser's Elements and Network tool inside the VS Code editor and use it to fix styling, layout, and CSS issues with your site.</h4>
+**Show the browser's Elements and Network tool inside the Visual Studio Code editor and use it to fix CSS issues with your site and inspect the Network.**
```
Comment (vidorteg) [inline README.md:4]: ...inspect the network requests?

Comment (codepo8) [reply README.md:4]: Or "network activity".

**Review by mliao95 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -65,20 +100,20 @@ module.exports = {
 
 With source maps enabled, you may also need to configure the extension settings/launch.json config to add customized paths between your runtime urls and your workspace paths, see [Sourcemaps](#sourcemaps) for more information.
 
+#### Debug Configuration
 
-### Debug Configuration
-You can launch the Microsoft Edge Tools extension like you would a debugger, by using a `launch.json` config file. However, Microsoft Edge Tools isn't a debugger and so any breakpoints set in VS Code won't be hit, you can of course use a different debug extension instead and attach the Microsoft Edge Tools extension once debugging has started.
+You can launch the Microsoft Edge Tools extension like you would a debugger, by using a `launch.json` config file. However, Microsoft Edge Tools isn't a debugger and so any breakpoints set in Visual Studio Code won't be hit, you can use a different debug extension instead and attach the Microsoft Edge Tools extension once debugging has started.
```
Comment (vidorteg) [inline README.md:105]: my 2c:
`...like you would a debugger, by using a `launch.json` config file` 
this is not totally clear to me :), maybe ellaborating a little more on this?

`However, Microsoft Edge Tools isn't a debugger and so any breakpoints set in Visual Studio Code won't be hit, you can use a different debug extension instead and attach the Microsoft Edge Tools extension once debugging has started.`
what about: 
`Microsoft Edge Tools works great when paired with [Debugger for Microsoft Edge](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-edge), you can use the first one to design your frontend and the latter to debug your code and set breakpoints`

File: README.md [hunk 4]
```diff
@@ -1,46 +1,81 @@
...
 One of the features of the Elements tool is that it can show you what file applied the styles and event handlers for a given node.
 
 ![Microsoft Edge Tools - Links](links.png)
 
-The source files for these applied styles and attached event handlers appear in the form of links to a url specified by the browser. Clicking on one will attempt to open that file inside the VS Code editor window. Correctly mapping these runtime locations to actual files on disk that are part of your current workspace, may require you to enable source maps as part of your build environment.
+The source files for these applied styles and attached event handlers appear in the form of links to a url specified by the browser. Clicking on one will attempt to open that file inside the Visual Studio Code editor window. Correctly mapping these runtime locations to actual files on disk that are part of your current workspace, may require you to enable source maps as part of your build environment.
 
 An example webpack configuration for sass and typescript is given below:
+
 ```javascript
```
Comment (vidorteg) [inline README.md:79]: I have not tried this but what about something more basic like:
```
"vscode-edge-devtools.pathMapping": {
            "/": "${workspaceFolder}",
            "/css":"${workspaceFolder}/customCss"
        }
```

File: README.md [hunk 5]
```diff
@@ -1,46 +1,81 @@
...
+
+You can enable the Network Pane to inspect any network request of the attached browser. To do this, change the setting and restart the extension.
+
+![Example how to turn on the network pane](edge-devtools-for-vscode-settings-network.png)
+
+You can see an example of the change in the following screencast. After restart, you get an extra tab with network functionality.
+
+![Example how to turn on the network pane](basic_usage(network_and_headless).gif)
+
+#### Turning on Headless Mode
```
Comment (vidorteg) [inline README.md:49]: I like this section being here for visibility but probably makes more sense at the bottom. I don't have a strong opinion on this so I'm fine with either.

**Review by vidorteg — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -1,46 +1,81 @@
...
+# Microsoft Edge Developer Tools for Visual Studio Code
 
-<h4 align="center">Show the browser's Elements and Network tool inside the VS Code editor and use it to fix styling, layout, and CSS issues with your site.</h4>
+**Show the browser's Elements and Network tool inside the Visual Studio Code editor and use it to fix CSS issues with your site and inspect the Network.**
 
-A VS Code extension that allows you to use the browser's Elements and Network tool from within the editor. The DevTools will connect to an instance of Microsoft Edge giving you the ability to see the runtime HTML structure, alter layout, fix styling issues, and view network requests. All without leaving VS Code.
+A Visual Studio Code extension that allows you to use the browser's Elements and Network tool from within the editor. The DevTools will connect to an instance of Microsoft Edge giving you the ability to see the runtime HTML structure, alter layout, fix styling issues, and view network requests. All without leaving Visual Studio Code.
 
-**Note**: This extension only supports Microsoft Edge
+**Note**: This extension _only_ supports Microsoft Edge
```
[the remark this answers is not in the document]
Comment (codepo8) [reply README.md:8]: fixed

File: README.md (hunk 3 above)
Comment (codepo8) [reply README.md:105]: Great re-write, done. Let's do the explanation of the launch.json file next, but I really want this to get out today as the legal names for VS Code need to be in there.

File: README.md (hunk 4 above)
Comment (codepo8) [reply README.md:79]: Let's try it and add it once we verified it.

**Pull request merged**